src/main/resources/application.yaml

```diff
@@ -2,52 +2,37 @@ server:
   port: ${PORT:4097}
 
 management:
-  health:
-    db:
-      enabled: true
-    defaults:
-      enabled: true
-    diskspace:
-      enabled: true
-    probes:
-      enabled: true
-  endpoint:
-    health:
-      enabled: true
-      cache:
-        time-to-live: 5s
-      show-details: "always"
-      group:
-        readiness:
-          include: "db"
-  security:
-    enabled: true
   endpoints:
-    health:
-      sensitive: true
     web:
-      base-path: /
       exposure:
-        include: 'health'
+        include: health,info,data-extract
```

Comment:
Can we drop `data-extract` from here

Reply:
removed

Reply:
Resolved in a264b1e0bde5de7e30b8c98f7c1687bfce0443af.